core/src/test/scala/unit/kafka/server/FetchRequestMaxBytesTest.scala

```diff
@@ -58,11 +58,13 @@ class FetchRequestMaxBytesTest extends BaseRequestTest {
     array
   }
 
+  @BeforeEach
```

Comment:
Does JUnit 5 not look for the annotation in superclasses? That is, is this for readability or is it required for behavior?

Reply:
yep, that is a behavior change. see  https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/BeforeEach.html

`@BeforeEach methods are inherited from superclasses as long as they are not overridden. Furthermore, @BeforeEach methods from superclasses will be executed before @BeforeEach methods in subclasses.`